.github/workflows/release-test.yml

```
@@ -88,14 +88,14 @@ jobs:
         ssh -oStrictHostKeyChecking=accept-new \
           ${IOTLAB_USER}@lille.iot-lab.info exit
     - name: Checkout Release-Specs
-      uses: actions/checkout@v2
+      uses: actions/checkout@master
```

Would this work without adding a version  (default to Head)?
```suggestion
      uses: actions/checkout
```

seems like github advices against using branches or no version (but the advice sound like no @ would result in HEAD)

or 

```suggestion
      uses: actions/checkout@main
```

---

They got a job that auto updates main to the latest tag https://github.com/actions/checkout/blob/main/.github/workflows/update-main-version.yml